docker-compose.yml

```diff
@@ -54,16 +54,8 @@ services:
     build:
       context: docker/images/fakesentry
     image: local/eliot_fakesentry
-    ports:
-      - "8090:8090"
     command: run --host 0.0.0.0 --port 8090
 
   # https://hub.docker.com/r/hopsoft/graphite-statsd/
   statsd:
     image: hopsoft/graphite-statsd:latest
-    ports:
-      - "8081:80"
```

Comment:
We need at least this one so we can use Grafana to look at metrics. It can be on a different port, but it should be exposed somehow.

Reply:
I'll make it configurable with env vars, and default to the existing port